setup.sh

```diff
@@ -116,16 +119,15 @@ mkdir -p /opt/ioi/store/screenshots
 mkdir -p /opt/ioi/store/submissions
 mkdir -p /opt/ioi/config/ssh
 
-# Latest as of 2023-05-20
-aria2c -x 4 -d /tmp -o cpptools-linux.vsix "https://github.com/microsoft/vscode-cpptools/releases/download/v1.15.4/cpptools-linux.vsix"
-wget -O /tmp/vscodevim.vsix "https://github.com/VSCodeVim/Vim/releases/download/v1.25.2/vim-1.25.2.vsix"
-rm -rf /tmp/vscode
```

Comment:
This is useful for testing. It was not probably an accident. This change is unrelated to the PR.